misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt

```diff
@@ -16,25 +16,27 @@ import java.nio.file.Paths
 abstract class PathFeatureAssertionsSpec(
     parentFeature: Feature0<Path, Path>,
     parent: Fun1<Path, Expect<Path>.() -> Unit>,
+    fileName: Feature0<Path, String>,
```

Comment:
please rename to `fileNameFeature` (see for instance fileNameWithoutExtensionFeature)
and add `fileName : Fun1<....>` (see fileNameWithoutExtension)